Sources/Sentry/SentryFileIOTracker.m

```diff
@@ -194,11 +194,9 @@ - (BOOL)measureNSFileManagerCreateFileAtPath:(NSString *)path
     }
 
     __block id<SentrySpan> ioSpan;
+    NSString *spanDescription = [self transactionDescriptionForFile:path fileSize:size];
     [SentrySDK.currentHub.scope useSpan:^(id<SentrySpan> _Nullable span) {
-        ioSpan = [span startChildWithOperation:operation
-                                   description:[self transactionDescriptionForFile:path
-                                                                          fileSize:size]];
-        ioSpan.origin = origin;
+        ioSpan = [span startChildWithOperation:operation description:spanDescription];
```

Comment:
`m`: Please add a comment to keep the logic here to a minimum, because of the issue we just had and reference the issue.